src/App.js

```diff
@@ -1,26 +1,19 @@
 import React from "react";
-import { Router } from "@reach/router";
-import { Layout } from "antd";
-
-import Header from "./components/header/Header";
-import Footer from "./components/footer/Footer";
 import { Home, Login } from "./pages";
-
-const { Content } = Layout;
+import { BrowserRouter, Route, Switch } from "react-router-dom";
```

Comment:
I prefer to use `@reach/router` instead of `react-router-dom`. You can refer to this post: https://reacttraining.com/blog/reach-react-router-future/

Unless you have strong opinion on using `react-router-dom`, I open to your suggestion.

Reply:
Tbh, i just use it because i always use it. I will try change it